src/platforms/java/common/gradle.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Gradle
+sidebar_order: 100
+description: "Learn about using the Sentry Gradle Plugin."
+---
+
+The [Sentry Gradle Plugin](https://github.com/getsentry/sentry-android-gradle-plugin) is an addition to the main Java SDK and offers
+seamless integration with the Gradle build system. It supports the report of external dependencies of the application as part of events.
```

Comment:
Can we rephrase it somehow so it's open for adding new features to it? Dependency report generation is just of the features, so I'd like to phrase it somehow to leave room for future features, because the plugin is capable of more things.

Reply:
What about?
```suggestion

The [Sentry Gradle Plugin](https://github.com/getsentry/sentry-android-gradle-plugin) is an addition to the main Java SDK and offers
seamless integration with the Gradle build system. It supports the reporting of your application's external dependencies as part of events.
```

Reply:
Just reminder that whatever phrasing we land on here needs to also be applied [here](https://github.com/getsentry/sentry-docs/pull/5712/files#diff-f05ad6205f886c7271d791282e44b2b79dc36f05e08c1784e9a31c0025b3ba7cR76).